governance/votes/Executive vote - November 29, 2023.md

```diff
@@ -0,0 +1,146 @@
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+	
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+- PSM-GUSD-A will be removed from the Cash Stablecoin list by setting the [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) to zero and removing the vault from the [Debt Ceiling Instant Access (autoline)](https://manual.makerdao.com/module-index/module-dciam).
+- Coinbase Custody (RWA014-A) [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be increased to 1.5 billion.
+- WBTC-A, WBTC-B, and WBTC-C [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased by 0.07%.
+- The [hop (frequency)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1) and [bump (lot size)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-3) SBE parameters will be changed to optimize for cost efficiency.
+- A RWA Foundation Dao Resolution will be approved.
+- A transfer will be made to cover Andromeda Legal expenses.
+- MKR and DAI Vest Streams for Steakhouse and TechOps will be Ceased.
+- Ecosystem Facilitator Vest Streams will be set up.
+- A Spark Protocol proxy-spell will be excecuted to address a identified discrepancy in the DAI Market.
+
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Remove PSM-GUSD-A from the Cash Stablecoin list
+
+*Authorization*: [Ecosystem Approval](http://forum.makerdao.com/t/stability-scope-parameter-changes-7/22882/2)\
+*Proposal*: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-7/22882#reduce-psm-gusd-a-debt-ceiling-8) "Stability Scope Parameter Changes"
+
+If this executive proposal passes, in line with the recent amendments in the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#7-2-1-3-1a), PSM-GUSD-A will be removed from the Cash Stablecoin list by performing the following actions:
+
+- The [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be set to *zero*.
```

Comment:
**Non Blocker**

nit: we prefer numerical "0" since it matches the code. https://github.com/makerdao/spells-mainnet/pull/372/files#diff-c9c3998420c475f01de841ca5283ea49ce6c99080a8c6a3fdae77b097bef6778R126-R127

```suggestion
- The [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be set to *0*.
```